odrive-motor/src/odriveCAN.py

```diff
@@ -197,9 +201,9 @@ async def is_powered(self, extra: Optional[Dict[str, Any]] = None, **kwargs) ->
 
     async def is_moving(self) -> bool:
         for msg in bus:
-            if msg.arbitration_id == ((self.nodeID << 5) | db.get_message_by_name('Get_Encoder_Estimates').frame_id):
-                current_vel = db.decode_message('Get_Encoder_Estimates', msg.data)['Vel_Estimate']
-                if current_vel > 0:
+            if msg.arbitration_id == ((self.nodeID << 5) | 0x009):
+                estimates = db.decode_message('Get_Encoder_Estimates', msg.data)
+                if abs(estimates['Vel_Estimate']) > 0.1:
```

Comment:
nice catch!

Reply:
actually it should probably be zero. this was something I tried when I was trying to figure out why the axis state wasn't changing back to idle